### PR TITLE
Added tooltips to all Icon buttons

### PR DESCRIPTION
**Pull request by mananjadhav**
@iwiznia Can you please review this?

### Details
Wrapped all TouchableOpacity + Icon components with Tooltip.

### Fixed Issues

$ #4499 

### Tests
1. Verified tooltips for IOUModal, Search, ReportActionCompose, AttachmentView, VideoChat.
2. Verified Tooltips for Web and Desktop
3. Tested for no impact on Mobile Web, iOS and Android

### QA Steps
1. Need to verify if all the icon buttons are covered. (exclude Green FAB)
2. Need to verify if any tooltip has any alignment issue.
3. Added tooltips for icon buttons in  IOUModal, Search, ReportActionCompose, AttachmentView, VideoChat.

### Tested On

- [x] Web
- [ ] Mobile Web
- [x] Desktop
- [ ] iOS
- [ ] Android

#### Web

tooltip-icon-close-download png
tooltip-icon-close-web
tooltip-icon-emoji-web
tooltip-icon-pin-web
tooltip-icon-search-web
tooltip-icon-unpin-web
tooltip-icon-vc-web

#### Mobile Web
NA

#### Desktop

tooltip-icon-close-desktop
tooltip-icon-download-desktop
tooltip-icon-emoji-desktop
tooltip-icon-pin-desktop
tooltip-icon-search-desktop
tooltip-icon-unpin-desktop
tooltip-icon-vc-desktop

#### iOS
NA

#### Android
NA

### CODE REVIEW

**Comment by mananjadhav** [comment]
@iwiznia PR raised.

Need help:
1. to confirm coverage. I'd search all `Icon` across the codebase and updated all relevant pressable icon
2. Translations in Spanish.
3. I haven't added Tooltips to the following buttons: 
       a. Main Send button in the Chat Input Box
       b. Green FAB on the LHN
       c. + button that opens up "Add attachment" option

Let me know if you feel tooltips are required for those too.

File: src/languages/es.js [hunk 1]
```diff
@@ -36,6 +36,10 @@ export default {
         contacts: 'Contactos',
         recents: 'Recientes',
         close: 'Cerrar',
+        download: 'Descargar',
+        pin: 'Alfiler',
```
Comment (iwiznia) [inline src/languages/es.js:40]:
```suggestion
        pin: 'Anclar',
```

File: src/languages/es.js [hunk 2]
```diff
@@ -36,6 +36,10 @@ export default {
...
+        unPin: 'Desprender',
+        back: 'De vuelta',
```
Comment (iwiznia) [inline src/languages/es.js:42]:
```suggestion
        back: 'Volver',
```

File: src/languages/es.js [hunk 3]
```diff
@@ -36,6 +36,10 @@ export default {
         contacts: 'Contactos',
         recents: 'Recientes',
         close: 'Cerrar',
+        download: 'Descargar',
+        pin: 'Alfiler',
+        unPin: 'Desprender',
```
Comment (iwiznia) [inline src/languages/es.js:41]:
```suggestion
        unPin: 'Desanclar',
```

File: src/languages/es.js [hunk 4]
```diff
@@ -101,6 +105,7 @@ export default {
         nameEmailOrPhoneNumber: 'Nombre, email o número de teléfono',
     },
     videoChatButtonAndMenu: {
+        tooltip: 'Chat de video',
```
Comment (iwiznia) [inline src/languages/es.js:108]:
```suggestion
        tooltip: 'Videollamada',
```

**Review by iwiznia — CHANGES_REQUESTED** [review]

**Comment by iwiznia** [comment]
> I haven't added Tooltips to the following buttons:

Why not?

**Comment by iwiznia** [comment]
Oh, also can you make sure to test in the other platforms? Even if it is to check there are no regressions and buttons look/work properly

**Comment by mananjadhav** [comment]
I'll update the translation changes.

> Why not?
One of them was a primary action. Other is a menu. I can add. Can you help with the translations for these two buttons ?
image


>  Oh, also can you make sure to test in the other platforms? Even if it is to check there are no regressions and buttons look/work properly
I've checked the other platforms - Mobile Web, iOS, and Android.

**Comment by iwiznia** [comment]
> One of them was a primary action. Other is a menu. I can add. Can you help with the translations for these two buttons ?

Sure. Let me tag the @Expensify/marketing team so they can check the copy (all new copy is [in this file](https://github.com/Expensify/App/pull/4661/files#diff-22037c1e4dbdea8803b450b3fce06e6c475762a31ecf84b34dc264ba0e443637R39), they are the green lines) and give us copy for the tooltips of these buttons:
a. Main Send button in the Chat Input Box
b. Green FAB on the LHN
c. + button that opens up "Add attachment" option

I can then translate the new ones for you.

**Comment by mananjadhav** [comment]
@iwiznia One quick question, what are your thoughts on opening the FAB menu and the + icon menu on hover? rather than adding Tooltip?

**Comment by iwiznia** [comment]
I am personally not quite fond of things that open on hover. In any case that's just one opinion, you can propose and discuss changes like that one in the slack channel.

**Comment by mananjadhav** [comment]
PR Updated with suggested translations and tooltip for send button.

**Comment by iwiznia** [comment]
b. Green FAB on the LHN
c. + button that opens up "Add attachment" option

We are still waiting for those, right?

**Comment by mananjadhav** [comment]
Yes waiting.

File: src/languages/es.js [hunk 5]
```diff
@@ -36,6 +36,10 @@ export default {
         contacts: 'Contactos',
         recents: 'Recientes',
         close: 'Cerrar',
+        download: 'Descargar',
+        pin: 'Anclar',
```
Comment (marklouisdeshaun) [inline src/languages/es.js:40]: I would suggest using `Fijar`. This is the terminology used in popular apps like WhatsApp and Twitter.

File: src/languages/es.js [hunk 6]
```diff
@@ -36,6 +36,10 @@ export default {
...
+        unPin: 'Desanclar',
```
Comment (marklouisdeshaun) [inline src/languages/es.js:41]: I would suggest using `Desfijar`. This is the terminology used in popular apps like WhatsApp and Twitter.

**Comment by mananjadhav** [comment]
@marklouisdeshaun Thanks, Mark. Will update. Can you help with the copy of the other two buttons? Floating Action Button in the sidebar and + button to open Add Attachments in the chat?

Screenshot 2021-08-16 at 5 25 56 PM

**Review by aldo-expensify — COMMENTED** [review]
I tested this locally and seemed to work fine.
I agree with the proposed changes on the spanish translations.

**Comment by mananjadhav** [comment]
@iwiznia @marklouisdeshaun Left with the copy & translation for the + action on ReportActionCompose.  Can you help with that to close it at the earliest?

**Review by iwiznia — DISMISSED** [review]

**Comment by mananjadhav** [comment]
I've set the labels as `Actions` and `Acción` right now.

**Review by iwiznia — DISMISSED** [review]

**Comment by mananjadhav** [comment]
> @iwiznia @marklouisdeshaun Left with the copy & translation for the + action on ReportActionCompose. Can you help with that to close it at the earliest?

@marklouisdeshaun Can you help me with the copy of this button? So that this can be moved to closure?

**Review by iwiznia — APPROVED** [review]

**Comment by botify** [comment]
This has been deployed to production and is now subject to a 7-day regression period.
If no regressions arise, payment will be issued on 2021-09-08.  :confetti_ball:

**Comment by eVoloshchak** [comment]
Leaving a note here that the PR has caused a regression: https://github.com/Expensify/App/issues/14043
The 'close' and 'download' tooltips are slightly misaligned 
image

**Pull request merged**